applications/matter_bridge/README.rst

```diff
@@ -0,0 +1,315 @@
+Configuration files are provided for different build types and they are located in the application root directory.
+
+The :file:`prj.conf` file represents a ``debug`` build type.
+Other build types are covered by dedicated files with the build type added as a suffix to the ``prj`` part, as per the following list.
+For example, the ``release`` build type file name is :file:`prj_release.conf`.
+If a board has other configuration files, for example associated with partition layout or child image configuration, these follow the same pattern.
+
+.. include:: /config_and_build/modifying.rst
+   :start-after: build_types_overview_start
+   :end-before: build_types_overview_end
+
+Before you start testing the application, you can select one of the build types supported by the sample.
+This sample supports the following build types:
+
+* ``debug`` -- Debug version of the application.
+  You can use this version to enable additional features for verifying the application behavior, such as logs.
+* ``release`` -- Release version of the application.
+  You can use this version to enable only the necessary application functionalities to optimize its performance.
+
+.. note::
+    `Selecting a build type`_ is optional.
+    The ``debug`` build type is used by default if no build type is explicitly selected.
+
+User interface
+**************
+
+.. include:: ../../samples/matter/lock/README.rst
+    :start-after: matter_door_lock_sample_led1_start
+    :end-before: matter_door_lock_sample_led1_end
+
+Button 1:
+     If pressed for six seconds, it initiates the factory reset of the device.
+     Releasing the button within the six-second window cancels the factory reset procedure.
+
+Button 2:
+     Enables Bluetooth LE advertising for the predefined period of time (15 minutes by default), and makes the device discoverable over Bluetooth LE.
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_jlink_start
+    :end-before: matter_door_lock_sample_jlink_end
+
+.. _matter_bridge_cli:
+
+Matter CLI commands
+===================
+
+You can use the following commands to control the bridge device:
+
+::
+
+   matter_bridge add <device_type> [node_label]
+
+This command adds a new bridged device to the bridge.
+In this command:
+
+* ``<device_type>`` is a Matter device type to use for the bridged device.
+  The argument is mandatory and accepts the following values:
+
+   * ``256 - On/Off Light.
+   * ``770`` - Temperature Sensor.
+   * ``775`` - Humidity Sensor.
+
+* ``[node label]`` is a node label for the bridged device.
+  The argument is optional and you can use it to better personalize the device in your application.
+   
+For example:
+
+.. parsed-literal::
+   :class: highlight
+
+   uart:~$ matter_bridge add 256 "Kitchen Light"
+
+::
+
+   matter_bridge remove <bridged_device_endpoint_id>
+
+This command removes a new bridged device from the bridge.
```

Comment:
I would drop "new" here, because this would suggest you can only remove the last added bridge.